src/oauth2.rs

```diff
@@ -210,8 +258,8 @@ impl Spotify {
             .post_form(auth_urls::TOKEN, Some(&head), payload)
             .await?;
         let mut tok = serde_json::from_str::<Token>(&response)?;
-        tok.expires_at = Some(datetime_to_timestamp(tok.expires_in));
-
+        tok.expires_at = Utc::now()
```

Comment:
This is a bit confusing, what are you trying to do?

Reply:
`Rspotify` use two fields to represent the expiration information, the `expires_in` field is used to store the time period for which the access token is valid(for example, 3600s), then the `expires_at` field is used to store the expiry datetime. For example, the current time is `2021-02-15 00:00:00` in Utc timezone, then the `expires_in` is `3600s`, then expiry datetime `expires_at` is `2021-02-15 01:00:00`. The token is valid if it's before `2021-02-15 01:00:00`.

If you are confusing about this field, probably we need a better, self-explanatory name for this field instead of `expires_at`.

Reply:
Sorry, what I meant was that I didn't understand these two lines of code. They seem a bit obfuscated in my opinion, why is `.checked_add_signed(tok.expires_in);` not possible?

Reply:
Yep, the original solution is impossible, since the required parameter type of `checked_add_signed ` is `chrono::Duration`, the type of `expires_in` is `std::time::Duration`.

So I just change the type of `expires_in` to `chrono::Duration`, it works.

Reply:
fixed: [`5893ed7` (#185)](https://github.com/ramsayleung/rspotify/pull/185/commits/5893ed770caef555dc466e865f24488999bdc5ee)